src/Entity/IBuyable.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Entity;
+
+interface IBuyable
```

Comment:
the name does not really fit the stuff it got.
buyable in my opinion should only rely on something like the price. or the type of product to set the VAT. but not the id, name or categories.